plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -260,20 +260,27 @@ private AppEngineDeployOutput getDeployOutput() throws CoreException {
 
   @VisibleForTesting
   public String getDeployedAppUrl(AppEngineDeployOutput deployOutput) {
-    String project = deployConfiguration.getProject();
     boolean promoting = deployConfiguration.getPromote();
     String version = deployOutput.getVersion();
     String service = deployOutput.getService();
+    String projectId = deployOutput.getProject();
     boolean usingDefaultService = DEFAULT_SERVICE.equals(service);
 
+    String domain = ".appspot.com";
+    int colon = projectId.indexOf(':');
+    if (colon >= 0) {
+      domain = ".googleplex.com";
+      projectId = projectId.substring(colon + 1);
+    }
+    
     if (promoting && usingDefaultService) {
-      return "https://" + project + ".appspot.com";
+      return "https://" + projectId + domain;
```

Comment:
I find them easier to read as inline string literals rather than referring back to a constant.